.travis.yml

```diff
@@ -1,12 +1,20 @@
 language: node_js
-node_js:
-  - "7"
-
+matrix:
```

Comment:
Why did you add a matrix with one build?

Reply:
I was copying the yml configuration from the CLI https://github.com/resin-io/resin-cli/blob/master/.travis.yml
I think if we drop the matrix value we can get rid of the "can deploy" env var as well.

Reply:
I'm tempted to keep this here in case we want to test against multiple node versions in the future